sdk/python/feast/cli.py

```diff
@@ -389,5 +392,24 @@ def registry_dump_command(repo_path: str):
     registry_dump(repo_config)
 
 
+@cli.command("materialize")
+@click.argument("repo_path", type=click.Path(dir_okay=True, exists=True))
+@click.argument("start_ts")
+@click.argument("end_ts")
+@click.option(
+    "--views", "-v", help="Feature views to materialize", multiple=True,
+)
+def materialize_command(repo_path: str, start_ts: str, end_ts: str, views: List[str]):
+    """
+    Run a (non-incremental) materialization job to ingest data into the online store.
```

Comment:
Should this contain a full docstring for end users? Especially important to know how timestamps are provided.